README.md

```diff
@@ -52,3 +52,17 @@ e.g.:
 * `RUBYFMT_DISABLE_SZUSH=1`: disables the backend render queue writer,
   very useful for debugging, literally useless if you're not developing rubyfmt
   itself.
+
+## Editor Support
+
+### Visual Studio Code
+
```

Comment:
you should reference the shim script, and then I will merge this.

Reply:
Updated README ✅